### DISCUSSION TITLE
How to use new React UI?

### DISCUSSION THREAD

**Discussion by Igorgro** [prometheus/prometheus#11670] [Q&A]
### What did you do?

I want to use modern React UI of prometheus, but still getting classic UI. When i'm visiting for example `/graph`, it redirects me to `/classic/graph`.

### What did you expect to see?

I want to see new React UI

### What did you see instead? Under which circumstances?

I see that Prometheus redirects me to classic UI and there is no option to open React UI.

### System information

Debian Bullseye )with backports)

### Prometheus version

```text
prometheus, version 2.33.5+ds1 (branch: debian/sid, revision: 2.33.5+ds1-2~bpo11+1)
  build user:       team+pkg-go@tracker.debian.org
  build date:       20220820-12:29:59
  go version:       go1.18.5
  platform:         linux/amd64
```

**Comment by roidelapluie** [comment]
Debian does not package the new UI. Using the react UI currently requires you do download the UI or the binary from our GitHub downloads.
